package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@demos-europe/demosplan-ui",
   "version": "0.3.21",
   "license": "MIT",
-  "private": false,
```

Comment:
I don't know if we need that, but yarn kicks this line out automatically all the time.